app/assets/stylesheets/credits.scss

```diff
@@ -117,8 +107,16 @@
     @include themeable(color, theme-secondary-color, $medium-gray);
     padding: 25px 12px;
     font-size: 0.8em;
-    @include themeable(border-top, theme-container-border, 1px solid lighten($light-medium-gray, 7%));
-    @include themeable(border-bottom, theme-container-border, 1px solid lighten($light-medium-gray, 7%));
+    @include themeable(
```

Comment:
Why is this sometimes split into multiple lines and sometimes it's the other way: https://github.com/thepracticaldev/dev.to/pull/6207/files#diff-9f426ab011d6177be0c850da788ef089R49 ?